cli/main.go

```diff
@@ -131,7 +132,7 @@ func (c *DiceDBClient) Executor(in string) {
 
 		// Execute other commands
 		res, err := c.client.Do(ctx, toArgInterface(args)...).Result()
-		if err != nil {
+		if err != nil && !errors.Is(err, dicedb.Nil) {
```

Comment:
A proper nil message would be a proper way to handle.
handle the dicedb.nil case in a separate if block with a proper message.